### DISCUSSION TITLE
count?

### DISCUSSION THREAD

**Discussion by kulnor** [PostgREST/postgrest#2544] [Q&A] [answered]
Apologies is this is obvious but started to use PostgREST and, but goind through the documentation, but can't seem to find a way to perform a count / x-tabulation query. 
Something like:
`select foo, count(*) from sometable group by 1`
Is this supported?

**Comment by wolfgangwalther** [comment, marked as answer]
We are not supporting aggregation queries via query syntax, yet. This has been requested and discussed in #915.

Your best bet right now is creating a `VIEW` or a RPC-`FUNCTION` to do the same.
